content-services/7.4/support/index.md

```diff
@@ -81,7 +81,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Alfresco Search and Insight Engine 2.0.7 and later | Search and Insight Engine is compatible with Java 11 as long as you run Zeppelin in a Java 8 runtime. You can do this either in a VM or separate Java 8 based server. |
 | Alfresco Search Services 2.0.7 and later | |
 | Alfresco Federation Services 3.0 | |
-| Keycloak 21.1.2 | Use this Keycloak version and Alfresco Keycloak Theme 0.3.5 as an alternative to Identity Service 2.0.0. |
+| Keycloak 24.0.3 | Use this Keycloak version and Alfresco Keycloak Theme 0.3.5 as an alternative to Identity Service 2.0.0. |

```

Comment:
Looks good to me, cc. @dsibilio - a sanity check please? (_this updates the ACS 7.4.2 table_)

Reply:
IMO it's ok, I'm not sure if @vprovaggi prefers to list Keycloak 21.1.2 and Keycloak 24.0.3 separately as potential alternatives to eachother?

Reply:
_(as a personal preference, I'd reckon a single line mentioning both "Keycloak 24.0.3, 21.1.2" would be acceptable in this case)_